service/src/main/java/bio/terra/workspace/service/resource/referenced/ReferencedBigQueryDataTableAttributes.java

```diff
@@ -0,0 +1,32 @@
+package bio.terra.workspace.service.resource.referenced;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ReferencedBigQueryDataTableAttributes {
+  private final String projectId;
+  private final String datasetName;
+  private final String dataTableName;
+
+  @JsonCreator
+  public ReferencedBigQueryDataTableAttributes(
+      @JsonProperty("projectId") String projectId,
```

Comment:
I _think_ Jackson can figure this class out without these annotations, but they certainly don't hurt anything.

Reply:
I'm not totally sure on this either, but the documentation for `JsonCreator` claims it's required:

```
NOTE: when annotating creator methods (constructors, factory methods), method must either be:
- Single-argument constructor/factory method without JsonProperty annotation for the argument: if so, this is so-called "delegate creator", in which case Jackson first binds JSON into type of the argument, and then calls creator. This is often used in conjunction with JsonValue (used for serialization).
- Constructor/factory method where every argument is annotated with either JsonProperty or JacksonInject, to indicate name of property to bind to
```